data/json/speech.json

```diff
@@ -2789,6 +2789,132 @@
     "sound": "someone calling for their child.",
     "volume": 25
   },
+  {
+    "type": "speech",
+    "speaker": [ "mon_feral_maid_broom", "mon_feral_maid_candlestick", "mon_feral_maid_knife" ],
+    "sound": "\"Here!\"",
+    "volume": 40
+  },
+  {
+    "type": "speech",
+    "speaker": [ "mon_feral_maid_broom", "mon_feral_maid_candlestick", "mon_feral_maid_knife" ],
+    "sound": "\"Here it is!\"",
+    "volume": 40
+  },
+  {
+    "type": "speech",
+    "speaker": [ "mon_feral_maid_broom", "mon_feral_maid_candlestick", "mon_feral_maid_knife" ],
+    "sound": "\"Found you!\"",
+    "volume": 30
+  },
+  {
+    "type": "speech",
+    "speaker": [ "mon_feral_maid_broom", "mon_feral_maid_candlestick", "mon_feral_maid_knife" ],
+    "sound": "\"You are trapped!\"",
+    "volume": 30
+  },
+  {
+    "type": "speech",
+    "speaker": [ "mon_feral_maid_broom" ],
+    "sound": "\"Time to take out the trash!\"",
+    "volume": 30
+  },
+  {
+    "type": "speech",
+    "speaker": [ "mon_feral_maid_candlestick" ],
+    "sound": "\"Time to turn off the lights!\"",
+    "volume": 30
+  },
+  {
+    "type": "speech",
+    "speaker": [ "mon_feral_maid_knife" ],
+    "sound": "\"Food is ready!\"",
+    "volume": 30
+  },
+  {
+    "type": "speech",
+    "speaker": [ "mon_feral_maid_knife" ],
+    "sound": "\"Today's menu is… You!\"",
+    "volume": 30
+  },
+  {
+    "type": "speech",
+    "speaker": [ "mon_feral_maid_knife" ],
+    "sound": "\"Looks like meat is back on the menu!\"",
+    "volume": 30
+  },
+  {
+    "type": "speech",
+    "speaker": [ "mon_feral_maid_broom", "mon_feral_maid_candlestick", "mon_feral_maid_knife" ],
+    "sound": "\"Come here!\"",
+    "volume": 30
+  },
```

Comment:
Idea: Add another one with "Lord, an intruder!" or something. Very specifically calling for their lords as your PR description indicates.

Reply:
I was actually lacking in imagination so I forgot to add one like that! I will add a couple of those!